index.html

```diff
@@ -0,0 +1,293 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
FYI head and body tags are optional; compare to e.g. https://glitch.com/edit/#!/boom-bath?path=tabindex.html for the style I often use

Reply:
For a lot of the head stuff I just went with the Glitch defaults, I'll go through and change to this style